Lib/tests/constants/PetConstants.test.ts

```diff
@@ -0,0 +1,55 @@
+import { describe, expect, it } from "vitest";
+import { PetConstants } from "../../src/constants/PetConstants";
+
+describe("PetConstants", () => {
+	describe("PROBABILITIES", () => {
+		it("should have each row sum to 1", () => {
+			for (const row of PetConstants.PROBABILITIES) {
+				const sum = row.reduce((acc, val) => acc + val, 0);
+				expect(sum).toBeCloseTo(1);
+			}
+		});
+
+		it("should have all rows with 5 columns", () => {
```

Comment:
maybe détailler mieux ce TU ("should have all rows with 5 columns *each corresponding to a rarity*")

Reply:
fixé